packages/core/ui/core/view-base/index.ts

```diff
@@ -619,8 +623,24 @@ export abstract class ViewBase extends Observable implements ViewBaseDefinition
 		}
 	}
 
+
+	_requetLayoutNeeded = false;
```

Comment:
@farfromrefug Could you just fix the spelling here:

```suggestion
	_requestLayoutNeeded = false;
```